src/stylesheets/components/_accordions.scss

```diff
@@ -67,7 +63,7 @@ $v-padding: $spacing-md-small;
 
   + .usa-accordion,
   + .usa-accordion-bordered {
-    margin-top: $spacing-small;
+    margin-top: units(1);
```

Comment:
It's worth investigating using the technique I'd mentioned on line 7 to use variables as before, except assign those vars system tokens